node-module-system/calculate.js

```diff
@@ -0,0 +1,32 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+function calculate() {
+  for (let i = 0; i < process.argv.length; i++) {
+    if (process.argv[3] === 'plus') {
+      return console.log(
+        'Result: ',
+        add(Number(process.argv[2]), Number(process.argv[4]))
```

Comment:
I like that you converted them all to number, but it would be nice to do it just once instead of repeating the same code over and over.